projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -1017,6 +1017,11 @@ public static function get_extensions_preset_for_variation( $preset_extensions_m
 			$preset_extensions = array_unique( array_merge( $preset_extensions, $production_extensions ) );
 		}
 
+		$single_extensions = isset( $preset_extensions_manifest->single )
```

Comment:
Make sure blocks defined in the `single` array of the manifest are loaded in the backend.